Cargo.toml

```diff
@@ -47,6 +47,7 @@ strum_macros = "0.21.1"
 regex = "1"
 ioctl-rs = { version = "0.2", optional = true }
 serde_cbor = "0.11.1"
+pancurses = { version = "0.16.1", features = ["win32"] }
```

Comment:
Will this break other platforms?

Reply:
This `"win32"` feature flag affects only Windows builds, it will be ignored on other platforms.